src/Raven.Server/Smuggler/Documents/ShardedDatabaseSmuggler.cs

```diff
@@ -40,7 +41,21 @@ public class ShardedDatabaseSmuggler : SmugglerBase
             long operationId,
             Action<IOperationProgress> onProgress = null, 
             CancellationToken token = default) : 
-            base(source, new MultiShardedDestination(source, databaseContext, handler, operationId), server.Server.Time, jsonOperationContext, options, result, onProgress, token)
+            this(source, new MultiShardedDestination(source, databaseContext, handler, operationId), jsonOperationContext, databaseRecord, server, options, result, onProgress, token)
```

Comment:
let's have a single ctor, and pass different destination from the caller

Reply:
done